const.js

```diff
@@ -0,0 +1,66 @@
+const roleEnum = ['admin', 'superadmin', 'manager', 'user'];
```

Comment:
Camel Case(preferred) or underscore case.
But it should be unified all through the app.